src/pollers/gateway.rs

```diff
@@ -30,16 +30,22 @@ use tonic::{
     codegen::{InterceptedService, Service},
     service::Interceptor,
     transport::{Certificate, Channel, Endpoint, Identity},
-    Status,
+    IntoRequest, Status,
 };
 use tower::ServiceBuilder;
 use url::Url;
 use uuid::Uuid;
 
 #[cfg(test)]
 use futures::Future;
+use std::collections::HashMap;
+use std::str::FromStr;
+use tonic::metadata::{MetadataKey, MetadataValue};
 
 static LONG_POLL_METHOD_NAMES: [&str; 2] = ["PollWorkflowTaskQueue", "PollActivityTaskQueue"];
+/// The server times out polls after 60 seconds. Set our timeout to be slightly beyond that.
+const LONG_POLL_TIMEOUT: Duration = Duration::from_secs(62);
+const OTHER_CALL_TIMEOUT: Duration = Duration::from_secs(30);
```

Comment:
What does "other" mean in this context? Anything other than a long poll?

Reply:
Yes, anything that isn't a long poll

Reply:
I would remove word other then as it's just a "normal" call timeout. Also isn't 30 sec a bit too much for a non long poll?